server/server.go

```diff
@@ -1204,9 +1206,7 @@ func (s *Server) campaignLeader() {
 	go s.member.KeepLeader(ctx)
 	log.Info("campaign pd leader ok", zap.String("campaign-pd-leader-name", s.Name()))
 
-	log.Info("setting up the global TSO allocator")
-	s.tsoAllocatorManager.SetUpAllocator(ctx, tso.GlobalDCLocation, s.member.GetLeadership())
-	defer s.tsoAllocatorManager.ResetAllocatorGroup(tso.GlobalDCLocation)
+	log.Info("initializing the global TSO allocator")
```

Comment:
How about moving it to before line 1215? The rest LGTM

Reply:
Done.